src/main/scala/is/hail/utils/Py4jUtils.scala

```diff
@@ -25,13 +25,12 @@ trait Py4jUtils {
     list
   }
 
-  def parseIntervalList(strs: java.util.ArrayList[String]): IntervalTree[Locus] =
-    IntervalTree(Locus.parseIntervals(strs.asScala.toArray), prune = true)
+  def parseIntervalList(strs: java.util.ArrayList[String]): IntervalTree[Locus, Unit] =
+    IntervalTree(Locus.parseIntervals(strs.asScala.toArray), noisy = true)
 
-  def makeIntervalList(intervals: java.util.ArrayList[Interval[Locus]]): IntervalTree[Locus] =
-    IntervalTree(intervals.asScala.toArray, prune = true)
+  def makeIntervalList(intervals: java.util.ArrayList[Interval[Locus]]): IntervalTree[Locus, Unit] =
+    IntervalTree(intervals.asScala.toArray, noisy = true)
 
-  // FIXME: don't use vector
   def arrayListToISeq[T](al: java.util.ArrayList[T]): IndexedSeq[T] = al.asScala.toIndexedSeq
```

Comment:
I still agree with don't use a vector.  I mean, at least we could do `new IndexedSeq[T] { def apply(i: Int) = al.get(i) ... }` which would be better than this.  A comment that mentions the reason we can't use an array is because we don't have the class tag might also be reasonable.  (Cotton, suggesting a comment?  Who hacked your Github account?)